<ui version="4.0" >
 <class>myQtAppDLG</class>
 <widget class="QDialog" name="myQtAppDLG" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>My Designer Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>20</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>80</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>20</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>160</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox1" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>70</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>110</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browse" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>20</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_do" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Do Something</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>260</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>110</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Yes or No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_about</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel" >
     <x>313</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
